monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -1145,6 +1145,12 @@ abstract class Observable[+A] extends Serializable { self =>
   final def doOnStart(cb: A => Unit): Observable[A] =
     self.liftByOperator(new DoOnStartOperator[A](cb))
 
+  final def doOnStartTask(cb: A => Task[Unit]): Observable[A] =
```

Comment:
Could you add scaladocs?
Similar to `doOnStart` but mentioning the difference and other variants (like `doOnNextAck` and others )

Reply:
sure, will do